webpack-build.config.js

```diff
@@ -27,12 +17,23 @@ webpackConfig.plugins.push(
 
   // Reference: http://webpack.github.io/docs/list-of-plugins.html#uglifyjsplugin
   // Minify all javascript, switch loaders to minimizing mode
-  new webpack.optimize.UglifyJsPlugin({
+  new webpack.optimize.UglifyJsPlugin( {
     compress: {
       warnings: false
     }
-  })
-
-);
+  } )
+];
+
+webpackConfig.plugins.push.apply( webpackConfig.plugins, webpackConfigPlugins );
```

Comment:
1. set engines to 5.* https://docs.npmjs.com/files/package.json#engines
2. use spread operator https://nodejs.org/en/docs/es6/#which-features-ship-with-node-js-by-default-no-runtime-flag-required
3. profit :)